llvm/unittests/DebugInfo/LogicalView/CMakeLists.txt

```diff
@@ -1,17 +1,22 @@
 set(LLVM_LINK_COMPONENTS
+  ${LLVM_TARGETS_TO_BUILD}
   AllTargetsDescs
   AllTargetsDisassemblers
   AllTargetsInfos
+  AsmPrinter
   DebugInfoLogicalView
   MC
   MCDisassembler
   TargetParser
   )
 
 add_llvm_unittest_with_input_files(DebugInfoLogicalViewTests
+  ../DWARF/DwarfGenerator.cpp
+  ../DWARF/DwarfUtils.cpp
```

Comment:
This is okay for now, but really these components should be moved into their own library so they can be shared between unittests.

Reply:
That is a good point. I will create a PR to record that request.